enumerable.rb

```diff
@@ -0,0 +1,76 @@
+module Enumerable
+  def my_each
```

Comment:
- [ ] **#my_each should return an Enumerator if no block is given**
Your Output: ` LocalJumpError: no block given (yield)`

- [ ] **enumerables #my_each when a block is given when self is an array should return the array itself after calling the given block once for each element in self**

- [ ] **enumerables #my_each when a block is given when self is a range should return the range itself after calling the given block once for each element in self.**
Your Output: 
```NameError: undefined local variable or method `length' for 5..50:Range```

- [ ] **enumerables #my_each when a block is given when self is a range behaves like #each calls the given block once for each element in self**
Your Output: 
```NameError: undefined local variable or method `length' for 5..50:Range```

- [ ] **enumerables #my_each when a block is given when self is a hash behaves like #each calls the given block once for each element in self**
```
expected: "[:a, 1][:b, 2][:c, 3][:d, 4][:e, 5]"
            got: ""
```

Reply:
Hello, I believe that the changes required are done.